src/apps/components/AppPage/AppPageNav.tsx

```diff
@@ -55,6 +63,18 @@ export const AppPageNav: React.FC<AppPageNavProps> = ({
         </Box>
       </Box>
       <Box display="flex" gap={4}>
+        <Button
+          whiteSpace={"nowrap"}
```

Comment:
**nitpick**
```suggestion
          whiteSpace="nowrap"
```

Reply:
Okay, I suggest you add this to ESLint to its automatically fixed and we don't need human intervention. I use this syntax because of a bug in Jetbrains (autocomplete for macaw fails without `{}`) - but it would be fixed in precommit if configured 

cc @andrzejewsky